sim/constants.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Simulator constants
+ * Pokemon Showdown - http://pokemonshowdown.com/
+ *
+ * @license MIT
+ */
+
+export const enum MoveEffectiveness {
```

Comment:
This doesn't work with sucrase

Reply:
> Compiles TypeScript code to JavaScript, removing type annotations and handling features like enums. Does not check types. Sucrase transforms each file independently, so you should enable the `isolatedModules` TypeScript flag so that the typechecker will disallow the few features like const enums that need cross-file compilation.

Which means we should probably turn on `isolatedModules` .

Reply:
! The emitted code seems valid though?

Reply:
Oh, I see. It's downgraded to a normal enum...